src/bp/ui-admin/src/Auth/index.ts

```diff
@@ -25,6 +27,18 @@ export function getActiveWorkspace() {
   return localStorage.getItem(WORKSPACE_KEY)
 }
 
+export function setChatUserAuth(auth?: ChatUserAuth) {
+  auth ? localStorage.setItem(CHAT_USER_AUTH_KEY, JSON.stringify(auth)) : localStorage.removeItem(CHAT_USER_AUTH_KEY)
```

Comment:
do we want to fallback to some other storage if localstorage is not supported

Reply:
Good point, we have a cookie backup in the studio, but not in the admin UI. I'll add that support in a different PR, since it's unrelated to this one